CMakeLists.txt

```diff
@@ -322,10 +322,10 @@ if(HAS_DIAG_COLOR_FLAG)
   endif()
 endif()
 
-option(TRAVIS_CI_BUILD "Travis/sourcehut CI, extra flags will be set" OFF)
```

Comment:
The sr.ht builds are currently defining `-DTRAVIS_CI_BUILD` and I think we would need @justinmk to change that.

Reply:
Never mind, just found where those are defined.